pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -609,6 +610,17 @@ impl RemoteTimelineClient {
         Ok(())
     }
 
+    /// Launch an index-file upload operation in the background, with only aux_file_policy flag updated.
+    pub(crate) fn schedule_index_upload_for_aux_file_policy_update(
+        self: &Arc<Self>,
+        last_aux_file_policy: Option<AuxFilePolicy>,
+    ) -> anyhow::Result<()> {
+        let mut guard = self.upload_queue.lock().unwrap();
+        let upload_queue = guard.initialized_mut()?;
+        upload_queue.last_aux_file_policy = last_aux_file_policy;
+        self.schedule_index_upload(upload_queue);
```

Comment:
I think this is unnecessary now because the child copies the flag, but it doesn't hurt.

We don't really have a test showing either way.

Reply:
decided to keep it for now... thinking about how to test it or what can break it but I'd like to leave this as an exercise for me next week :)